packages/wxt/src/types.ts

```diff
@@ -844,6 +844,38 @@ export type ResolvedPerBrowserOptions<T, TOmitted extends keyof T = never> = {
     : T[key];
 } & { [key in TOmitted]: T[key] };
 
+/**
+ * Firefox data collection permission types for personal data.
+ * See: https://extensionworkshop.com/documentation/develop/firefox-builtin-data-consent/#specifying-data-types
+ */
+export type FirefoxDataCollectionType =
+  | 'locationInfo'
+  | 'browsingActivity'
+  | 'websiteContent'
+  | 'websiteActivity'
+  | 'searchTerms'
+  | 'bookmarksInfo'
+  | 'healthInfo'
+  | 'contactInfo'
+  | 'socialInfo';
```

Comment:
I don't want to be responsible for maintaining this list of strings, I don't have the capacity to make quick changes to this list as Mozilla comes up with these things.

Adding this string union will show hints in your editor for all the string literals defined above, but won't throw an error if you type something else.

```suggestion
  | 'socialInfo'
  | string & {};
```